starlight/static/webpack-stats.json

```diff
@@ -0,0 +1 @@
+{"status":"done","chunks":{"main":[{"name":"main.js","path":"/code/starlight/static/build/main.js"}]}}
```

Comment:
This file should be in the `.gitignore` and not committed here, since it's a per-environment file.